project/.travis/before_install_test.sh.twig

```diff
@@ -14,6 +14,13 @@ fi
 composer self-update --stable
 sed --in-place "s/\"dev-master\":/\"dev-${TRAVIS_COMMIT}\":/" composer.json
 
+
+if [ ${TRAVIS_PHP_VERSION} '>=' '7.0' ]; then
+    {% if 'mongodb' in services %}
+    pecl install --force mongodb-stable
+    composer require "alcaeus/mongo-php-adapter" --ignore-platform-reqs --no-update
```

Comment:
I mean [this](https://github.com/sonata-project/dev-kit/blob/c5b400abff42957e8252c4390fc8423887df65cc/config/dev-kit.yml#L45) and the usage [here](https://github.com/sonata-project/dev-kit/blob/c5b400abff42957e8252c4390fc8423887df65cc/config/projects.yml).